ui/rewards/steps/CongratsStepContent.tsx

```diff
@@ -0,0 +1,124 @@
+  const { referralsQuery, rewardsConfigQuery } = useRewardsContext();
+
+  const registrationReward = rewardsConfigQuery.data?.rewards.registration;
+  const referralReward = Number(rewardsConfigQuery.data?.rewards.registration_with_referral) - Number(rewardsConfigQuery.data?.rewards.registration);
+  const refLink = `https://eth.blockscout.com?ref=${ referralsQuery.data?.code }`;
+  const shareText = `Just signed up for @blockscoutcom Merits program and got ${ registrationReward } merits!\n\nUse my referral link to get extra ${ referralReward } merits: ${ refLink }`; // eslint-disable-line max-len
+  return (
+    <>
+      <Flex
+        alignItems="center"
+        background="linear-gradient(254.96deg, #9CD8FF 9.09%, #D0EFFF 88.45%)"
+        borderRadius="md"
+        padding={ 2 }
+        mb={ 8 }
+      >
+        <Icon as={ meritsIcon } boxSize="60px" mb={ -1 }/>
+        <Skeleton isLoaded={ !rewardsConfigQuery.isLoading }>
+          <Text fontSize="30px" fontWeight="700" color="blue.700">
+            +{ rewardsConfigQuery.data?.rewards[ isReferral ? 'registration_with_referral' : 'registration' ] }
+          </Text>
+        </Skeleton>
+        { isReferral && (
+          <Flex alignItems="center" h="56px">
+            <Box w="1px" h="full" bgColor="whiteAlpha.800" mx={ 8 }/>
+            <Flex flexDirection="column" justifyContent="space-between">
+              { [
+                {
+                  title: 'Registration',
+                  value: registrationReward,
+                },
+                {
+                  title: 'Referral program',
+                  value: referralReward,
+                },
+              ].map(({ title, value }) => (
+                <Flex key={ title } alignItems="center">
+                  <Icon as={ meritsIcon } boxSize={ 8 } mb={ -0.5 }/>
+                  <Skeleton isLoaded={ !rewardsConfigQuery.isLoading }>
+                    <Text fontSize="sm" fontWeight="700" color="blue.700">
+                      +{ value }
+                    </Text>
+                  </Skeleton>
+                  <Text fontSize="sm" color="blue.700" ml={ 2 }>
+                    { title }
+                  </Text>
+                </Flex>
+              )) }
+            </Flex>
+          </Flex>
+        ) }
+      </Flex>
+      <Flex flexDirection="column" alignItems="flex-start" px={ 3 } mb={ 8 }>
+        <Flex alignItems="center" gap={ 2 }>
+          <Box w={ 8 } h={ 8 } p={ 1.5 } borderRadius="8px" backgroundColor="blue.50">
+            <IconSvg name="profile" boxSize={ 5 } color="blue.500"/>
+          </Box>
+          <Text fontSize="lg" fontWeight="500">
+            Referral program
+          </Text>
+        </Flex>
+        <Text fontSize="md" mt={ 2 }>
+          Receive a{ ' ' }
+          <Skeleton as="span" isLoaded={ !rewardsConfigQuery.isLoading }>
+            { Number(rewardsConfigQuery.data?.rewards.referral_share || 0) * 100 }%
```

Comment:
I think if it is zero then it doesn't make any sense.

Reply:
No, I mean the whole element should not be visible (maybe)

Reply:
Decided to display N/A